app/main/views.py

```diff
@@ -107,31 +107,16 @@ def search_resources():
     resources_as_dicts = Resource.get_resources_as_dicts(resources)
     return json.dumps(resources_as_dicts)
 
+
 @main.route('/get-associations/<int:resource_id>')
 def get_associations(resource_id):
     resource = Resource.query.get(resource_id)
-    associations = {}
-    if resource is None:
-        return json.dumps(associations)
-    for td in resource.text_descriptors:
-        associations[td.descriptor.name] = td.text
-    for od in resource.option_descriptors:
-        val = od.descriptor.values[od.option]
-        values = set()
-        # multiple option association values
-        if associations.get(od.descriptor.name):
-            curr = associations.get(od.descriptor.name)
-            curr.append(val)
-            values = set(curr)
-        else:
-            values.add(val)
-        associations[od.descriptor.name] = list(values)
-    return json.dumps(associations)
+    return json.dumps(Resource.get_associations(resource))
 
 
-@main.route('/pages/<pageName>')
-def render_page(pageName):
-    editable_html_obj = EditableHTML.get_editable_html(pageName)
+@main.route('/pages/<page_name>')
```

Comment:
Can we put these edits in a separate PR? Also this is causing a build error:
![image](https://user-images.githubusercontent.com/20306315/37981978-21df07ec-31bd-11e8-992d-ff27c29fbd67.png)
(Works if I recreate the DB)

Reply:
ok NVM about putting it in a separate PR, but is there a way to fix this BuildError without having to recreate the db

Reply:
Tbh this was just a style fix that bothered me, I just changed it back bc it's not super important!